sdk/ai/azure-ai-inference/samples/sample_chat_completions_from_input_prompty.py

```diff
@@ -0,0 +1,74 @@
+# ------------------------------------
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+# ------------------------------------
+"""
+DESCRIPTION:
+    This sample demonstrates how to get a chat completions response from
+    the service using a synchronous client, with the input in Prompty format
+    from a Prompty file. Prompty website: https://prompty.ai
+
+    This sample assumes the AI model is hosted on a Serverless API or
+    Managed Compute endpoint. For GitHub Models or Azure OpenAI endpoints,
+    the client constructor needs to be modified. See package documentation:
+    https://github.com/Azure/azure-sdk-for-python/blob/main/sdk/ai/azure-ai-inference/README.md#key-concepts
+
+USAGE:
+    python sample_chat_completions_from_input_prompty.py
+
+    Set these two environment variables before running the sample:
+    1) AZURE_AI_CHAT_ENDPOINT - Your endpoint URL, in the form 
+        https://<your-deployment-name>.<your-azure-region>.models.ai.azure.com
+        where `your-deployment-name` is your unique AI Model deployment name, and
+        `your-azure-region` is the Azure region where your model is deployed.
+    2) AZURE_AI_CHAT_KEY - Your model key (a 32-character string). Keep it secret.
+"""
+# mypy: disable-error-code="union-attr"
+# pyright: reportAttributeAccessIssue=false
```

Comment:
Why do we have to globally ignore? What line has the error?

Reply:
Thanks! It's copied from the other example, I'll remove them for now and only add the required ones when we work on linters